app/serializers/service_provider_serializer.rb

```diff
@@ -0,0 +1,43 @@
+class ServiceProviderSerializer < ActiveModel::Serializer
+  attributes(
+    :issuer,
+    :agency,
+    :friendly_name,
+    :metadata_url,
+    :acs_url,
+    :assertion_consumer_logout_service_url,
+    :cert,
+    :block_encryption,
+    #:return_to_sp_url,
+    #:logo,
+    #:attribute_bundle,
+    :updated_at,
+    :signature
+  )
+
+  def agency
+    object.name
+  end
+
+  def friendly_name
+    object.description
```

Comment:
Sorry that was a forward-thinking wonder (ex when we make a migration to add the other columns we need, should we consider renaming these existing columns in the DB too?)